### PR TITLE
Add Audiocraft snippet.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -366,6 +366,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			wildcard: { path: "*.sentis" },
 		},
 	},
+	"audiocraft-musicgen": {
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:369]: Libraries in this file are listed by alphabetical order. Would be good to keep the same (so moving this entry between `asteroid` and `bertopic`).

**Review by Wauplin — COMMENTED** [review]
Regarding the name `"audiocraft-musicgen"`, are audiocraft and musicgen two names related to the same thing/library?

I'm asking because the github project is [facebookresearch/audiocraft](https://github.com/facebookresearch/audiocraft) and at the moment there are [60 models](https://huggingface.co/models?other=musicgen) tagged a `"musicgen"`, [11 models](https://huggingface.co/models?other=audiocraft) tagged as `"audiocraft"` and [0 models](https://huggingface.co/models?other=audiocraft-musicgen) tagged as `"audiocraft-musicgen"` on the Hub. Adding an "official" tag in this PR is a good time to harmonize those (and maybe keep the most widely used?).
Just FYI, the entry key in the mapping -`audiocraft-musicgen` here- is the one that will be looked for in the modelcard metadata while the "pretty label" will be the one displayed on the Hub.

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -185,6 +185,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.mlx,
 		filter: true,
 	},
+	musicgen: {
+		prettyLabel: "MusicGen",
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:189]: Shouldn't this say audiocraft?

**Review by Wauplin — APPROVED** [review]
Looks good to me! I haven't tried the snippet myself but I trust you on this part :)

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -185,6 +185,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.mlx,
 		filter: true,
 	},
+	musicgen: {
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:188]: I think it's weird the library is `audiocraft` but the filter is `musicgen`. I would use `audiocraft` everywhere, that will scale better + we can then use this for other architectures as well (audiogen, encodec, etc).

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:188]: But there is no filter right? We explicitly mark `filter: false` - audiocraft can have quite a lot of different APIs for different models - magnet + audiogen.

**Comment by Vaibhavs10** [comment]
As discussed offline w/ @osanseviero - this now has all the code snippets in one master snippet. This covers all the use-cases for official checkpoints from audiocraft.

**Review by osanseviero — APPROVED** [review]
LGTM! Let's wait for a TS reviewer too

**Review by julien-c — APPROVED** [review]
TS code looks good to me!

**Review by pcuenca — APPROVED** [review]

**Pull request merged**